community/chromium/build

```diff
@@ -0,0 +1,122 @@
+#!/bin/sh -e
+
+kiss list musl && {
+    for patch in musl-*.patch; do
+        patch -p0 < "$patch"
+    done
+}
+
+for patch in chromium-*.patch; do
+    patch -p1 < "$patch"
+done
+
+# Avoid falling back to preprocessor mode when sources contain time macros
+kiss list ccache && export CCACHE_SLOPPINESS=time_macros
+
+# Force script incompatible with Python 3 to use /usr/bin/python2
+sed -i '1s|python$|&2|' third_party/dom_distiller_js/protoc_plugins/*.py
+
```

Comment:
Putting
```
mkdir -p third_party/node/linux/node-linux-x64/bin
ln -s /usr/bin/node third_party/node/linux/node-linux-x64/bin/node
```
up here will do the same thing as the `chromium-system-node.patch`. This is how Arch does it, and its a bit less fragile.

Reply:
Yeah that's how i was doing it earlier, but thought that the patch made the build file a little less ugly /messy :p, doesn't do much at all, just makes the python script return `/usr/bin/node`

Reply:
Fair enough. IMO putting it in the build script is cleaner, but its your package :)